### ISSUE TITLE
Add signal for process_tasks start

### ISSUE THREAD

**Issue by kipe** [arteria/django-background-tasks#240]
I'm suggesting addition of a new signal, such as `process_tasks_started`. The signal would be emitted by the `process_tasks` management command, and would serve as a signal to initialize repeated tasks.

The issue raises from multiple instances of Django running in parallel (multiple web workers, MQTT client etc). When a repeated task is started from `AppConfig.ready()`, it's done multiple times with the current setup. With a custom signal, the initialization of `process_tasks_started` the initialization could be tied to initialization of the process queue, at least reducing the possibility of triggering the task multiple times.

I'm planning on implementing this on my own fork, so if this is something to be considered, I'm willing to make a pull request.

**Comment by kipe** [comment]
This is now implemented in my own fork: https://github.com/kipe/django-background-tasks/commit/32109a87a60249aaf2785cdbbb06b97b4de26bea

As an example usage, the `signals.py` for one of my apps:
```python
from django.dispatch import receiver
from background_task.signals import process_tasks_started


@receiver(process_tasks_started)
def start_nordpool_fetching(**kwargs):
    from background_task.models import Task
    from raspautomation.plugin.nordpool.tasks import fetch_nordpool

    fetch_nordpool(repeat=Task.HOURLY)
```

**Comment by hodossy** [comment]
I had the same issue, but I am also running multiple instances of workers, and if I understand correctly that would mean that the issue would still remain, it would just be triggered by the workers and not Django.

In the end I have implemented a custom management command to trigger these repeated tasks and added it to my deployment pipeline.

**Comment by kipe** [comment]
Yeah, this doesn't solve that problem directly. For my own use I just added `remove_existing_tasks=True` to the `background` decorator, which seemed to fix the issue. Needs further testing though.

```python
@background(remove_existing_tasks=True)
def fetch_nordpool():
    logger.info('Task started')
```
